src/cmds/cloud/createThing.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-console */
+import yargs from 'yargs';
+import { Client } from '@cesarbr/knot-cloud-sdk-js';
+import options from './options';
+
+const createThing = (args) => {
+  const client = new Client({
+    protocol: args.protocol,
+    hostname: args.server,
+    port: args.port,
+    pathName: args.pathName,
+    id: args['client-id'],
+    token: args['client-token'],
+  });
+
+  client.on('ready', () => {
+    client.register({
+      type: 'knot:thing',
+      id: args.id,
+      name: args.name,
+    });
+  });
+  client.on('registered', (device) => {
+    console.log(device);
```

Comment:
I think it'd be better to return a well-stringified JSON since it remains readable and can be redirected to a file in order to be used on thing operations (I've added this option to all commands).